### DISCUSSION TITLE
Clipboard not working - MacOS 11.6 - M1 - UTM 3.0.4 - XP SP3

### DISCUSSION THREAD

**Discussion by Foiler-NZ** [utmapp/UTM#3736] [UTM Help (macOS)]
I can't get Clipboard to work between my Macbook Pro M1 MacOS 11.6 host and VM Windows XP SP3 32-bit running under UTM ver 3.0.4. 

I have installed different versions of SpiceTools and VDagent.exe  .. all to no avail. The starting point was spice-guest-tools-0.164.2.iso

Under a Windows 7 VM I am also running: the clipboard is fine, after installing from spice-guest-tools-0.164.2.iso. 

It seems that under Win XP the process for vdagent.exe will not run for more than about half a second .. whereas under W7 it can be stopped and started either as a process or from the installed service, and it is stable. The shared Clipboard seems to need this agent running .. I wonder why vdagent.exe will not run on my Win XP VM ? 

I have checked that the 32-bit version of vdagent.exe is installed .. there is no error message when running it, just stops after a fraction of a second .. whereas the 64-bit version throws up an appropriate error message.

**Comment by ilhamsj** [comment]
same here device M1 2020 with monterey OS 12.2.1, i install UTM VM ubuntu the sharing folder is working perfecly but not for clipboard and scroll wheel.

no desktop version server only

**Comment by nohaelgendy** [comment]
I have the same issue with my M1 and Ubuntu on UTM. Does anyone find a solution ??

**Comment by soopermodel** [comment]
same issue.
